client/spec/master_desk_spec.js

```diff
@@ -0,0 +1,116 @@
+
+'use strict';
+var openUrl = require('./helpers/utils').open;
+
+function MasterDesks() {
```

Comment:
Per Mugur feedback no (see above his comment).
I think we have to add on pages things that potentially are used in multiple tests, like workspace manipulation. Otherwise in time we will have a too big pages file like in the case of steps.